loadtests/loadtest.py

```diff
@@ -0,0 +1,18 @@
+from molotov import scenario
+
+
+HOMEPAGE = 'https://fx-breach-alerts.herokuapp.com'
+SCAN = 'https://fx-breach-alerts.herokuapp.com/scan'
+TEST_EMAIL_HASH = 'b84c4c03b2af4050ac2d3b105e58adf83fa5af05'
```

Comment:
Want to add a comment to mention that this hash is for test@example.com?

Reply:
Wait, how did you get that hash? I tried this, but get a different hash value:

```sh
$ echo "test@example.com" -n | shasum # 4235978ef5585c7910233bfa7b8aed61e96b3003
```

Reply:
Odd, even <kbd>$ node scripts/make-sha1-hashes.js</kbd> gives me an interestingly different result:

> You entered: [test@example.com], sha1 hash of lowercase: 567159d622ffbb50b11b0efd307be358624a26ee